src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -472,52 +512,181 @@ private void CreateRegionLinks()
                 if (connections.ContainsKey(connectionKey) || connections.ContainsKey(reverseConnectionKey))
                     continue;
 
-                var pathToAdjacent = GetLeastIntersectingPath(region, adjacent);
+                var (pathToAdjacent, startVertical, endVertical) = GetLeastIntersectingPath(region, adjacent);
 
-                connections.Add(connectionKey, pathToAdjacent);
+                connections.Add(connectionKey, (pathToAdjacent, startVertical, endVertical));
             }
         }
 
         AdjustPathEndpoints();
     }
 
+    private int AdjustPriority(int size1, int priority)
+    {
+        // Prioritize connections to the longer sides
+        priority += size1 * size1;
+
+        return priority;
+    }
+
+    private int GetIShapePathCharacteristics(int startRows, int startColumns, int endRows, int endColumns,
```

Comment:
Shouldn't the name be `GetIShapePathPriority`? As this returns the priority and not some other unspecified characteristic.

Also the the other "characteristics" methods seem to have this same problem with them.

Reply:
That is true but I wanted to be consistent in all these paths, but I will change it to priority